src/C/common.jl

```diff
@@ -20,29 +21,21 @@ const CPLE_AWSSignatureDoesNotMatch = 16
 
 # const CPLAssert = expr
 
-# begin enum CPLErr
-const CPLErr = UInt32
+# begin enum ANONYMOUS_1
+const ANONYMOUS_1 = UInt32
```

Comment:
Is it me, or do the generated enums look off? `CPLErr` should still be `UInt32`, rather than being rewritten as `ANONYMOUS_$i`. It's the same with the other enums as well.

Reply:
Yeah they are off. I added this code to deal with it when rewriting: https://github.com/visr/GDAL.jl/blob/d9cabd4295819b924bf9a753d0b2021880fba4eb/gen/rewriter.jl#L193-L199

But that's more of a patch to fix something that seems to be going wrong in Clang.jl. I realize however that this now makes the C submodule hardly usable, so let's try to see if we can fix this in Clang.jl before merging.

Reply:
`ANONYMOUS_$i` no longer occurs in our wrapping, due to a number of fixes in `visr/Clang.jl`, that still need to be upstreamed.
Enums are now wrapped as enums.